docs/markdown/Gnome-module.md

```diff
@@ -199,6 +199,13 @@ for a build target, you must add the generated header to the build
 target's list of sources to codify the dependency. This is true for
 all generated sources, not just `mkenums_simple`.
 
+The generated `.c` file `#include` all headers passed to `sources` keyword
```

Comment:
Maybe "The generated source file includes all headers passed to the `sources` keyword argument"

Reply:
done